lib/services/elasticsearch.js

```diff
@@ -1667,9 +1797,8 @@ class ElasticSearch extends Service {
               get: result.get // used by mUpdate to get the full document body
             });
           }
-
-          i++;
         }
+        /* end critical code section */
 
         return {
           items: successes, // @todo rename items to documents
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)